schema.sql

```diff
@@ -64,4 +64,12 @@ relationship, it should also keep track of the date of the visit.*/
 
 create table visits (animals_id INT, vets_id INT, date_of_visit date);
 ALTER TABLE visits ADD CONSTRAINT animals_fkey FOREIGN KEY (animals_id) REFERENCES animals(id);
-ALTER TABLE visits ADD CONSTRAINT vets_fkey FOREIGN KEY (vets_id) REFERENCES vets(id);
\ No newline at end of file
+ALTER TABLE visits ADD CONSTRAINT vets_fkey FOREIGN KEY (vets_id) REFERENCES vets(id);
+
+-- Add an email column to your owners table
+ALTER TABLE owners ADD COLUMN email VARCHAR(120);
+
+---PERFORMANCE IMPROVEMENT---
+CREATE index animals_id_index ON visits (animals_id);
+CREATE INDEX vets_id_index ON visits (vets_id);
+CREATE INDEX owners_email_index ON owners (id);
```

Comment:
- [ ] I see that you have added this code to improve performance but you are expected to provide very easy-to-follow screenshots of **before** performance is optimized **after** performance is optimized. The screenshots you have provided in this pull request contain only the **_before_** screenshots and not the **_after_**. if it does, then I cannot see it, kindy make it obvious like this 👇🏾 

- SELECT COUNT(\*) FROM visits where animal_id = 4;

Before
 
![image](https://user-images.githubusercontent.com/84156714/215564188-9796bff6-deca-4011-bf34-ffb4d646b92a.png)

After
![image](https://user-images.githubusercontent.com/84156714/215564222-f4347e41-9df3-4317-a3a1-e4dc03ffa711.png)


Please do the same for the other two problematic queries.